Documents/idea document 2/ideadocument2.md

```diff
@@ -7,8 +7,38 @@
 # Methods
 (Thom)
 
-# Speficications & Execution Plan
+# Specifications & Execution Plan
 
 # Expected Outcomes
 
 # Evaluation & Outlook
+
+## Evaluation
+In the section Specifications & Execution Plan a list of MoSCoW Specifications was presented.
+These metrics are a useful and structured manner to evaluate wether or not the application is working as intended.
```

Comment:
First you mention specifications, the next sentence they are metrics. I get it, but I think its a bit weird